### PR TITLE
refactor: Convert EventBuilder to TypeScript

### PR DESCRIPTION
**Pull request by ffflorian**

### CODE REVIEW

File: src/script/conversation/ConversationVerificationStateHandler.ts [hunk 1]
```diff
@@ -135,7 +135,10 @@ export class ConversationVerificationStateHandler {
   private checkChangeToVerified(conversationEntity: Conversation): boolean {
     if (this.willChangeToVerified(conversationEntity)) {
       const currentTimestamp = this.serverTimeHandler.toServerTimestamp();
-      const allVerifiedEvent = z.conversation.EventBuilder.buildAllVerified(conversationEntity, currentTimestamp);
+      const allVerifiedEvent = window.z.conversation.EventBuilder.buildAllVerified(
```
Comment (bennycode) [inline src/script/conversation/ConversationVerificationStateHandler.ts:138]: Window namespace

File: src/script/conversation/ConversationVerificationStateHandler.ts [hunk 2]
```diff
@@ -175,7 +178,12 @@ export class ConversationVerificationStateHandler {
       }
 
       const currentTimestamp = this.serverTimeHandler.toServerTimestamp();
-      const event = z.conversation.EventBuilder.buildDegraded(conversationEntity, userIds, type, currentTimestamp);
+      const event = window.z.conversation.EventBuilder.buildDegraded(
```
Comment (bennycode) [inline src/script/conversation/ConversationVerificationStateHandler.ts:181]: Window namespace

File: src/script/conversation/ConversationVerificationStateHandler.ts (hunk 1 above)
Comment (ffflorian) [reply src/script/conversation/ConversationVerificationStateHandler.ts:138]: When I remove the namespace here, the tests fail.

File: src/script/conversation/ConversationVerificationStateHandler.ts (hunk 2 above)
Comment (ffflorian) [reply src/script/conversation/ConversationVerificationStateHandler.ts:181]: When I remove the namespace here, the tests fail.

**Pull request merged**